src/ReflectionMethod.php

```diff
@@ -107,14 +108,20 @@ public function __toString()
             $this->isFinal() ? ' final' : '',
             $this->isStatic() ? ' static' : '',
             $this->isAbstract() ? ' abstract' : '',
-            join(' ', \Reflection::getModifierNames($this->getModifiers() & 1792)),
+            join(
+                ' ',
+                \Reflection::getModifierNames(
+                    $this->getModifiers() &
+                        (self::IS_PUBLIC | self::IS_PROTECTED | self::IS_PRIVATE)
```

Comment:
If I read the issue correctly, @Nitpick-CI wants `$this->getModifiers()` intended the same as `(self::IS_PUBLIC` which I was concerned may be easily mistaken as a second argument to `getModifierNames()`. Despite not liking overly long lines, I think getting rid of the line break is the best option.

As above, unambiguous options in my personal order of preference:

1. Remove the newline:
```php
            join(
                ' ',
                \Reflection::getModifierNames(
                    $this->getModifiers() & (self::IS_PUBLIC | self::IS_PROTECTED | self::IS_PRIVATE)
                )
            ),
```
2. Square up the two lines:
```php
            join(
                ' ',
                \Reflection::getModifierNames(
                    $this->getModifiers() &
                    (self::IS_PUBLIC | self::IS_PROTECTED | self::IS_PRIVATE)
                )
            ),
```
2. Leave them as-is:
```php
            join(
                ' ',
                \Reflection::getModifierNames(
                    $this->getModifiers() &
                        (self::IS_PUBLIC | self::IS_PROTECTED | self::IS_PRIVATE)
                )
            ),
```